pkg/remote/README.md

```diff
@@ -0,0 +1,59 @@
+# remote: push arbitrary trace level data to an influxdb instance
```

Comment:
nit: I'd prefer to call this package something a little more indicative of what it does. `remote` is quite a generic term. I'd be for calling it `trace` or perhaps `eventcollector`. Also If we expect most people to view the docs of this repo through godoc then I think it would be better if this was a doc.go file instead of a README.md.

Reply:
good idea, trace is a significant improvement https://github.com/celestiaorg/celestia-core/pull/970/commits/b51a4447915b8c69632015dd93c3e1689eeab39b

Reply:
also copied the content of the readme into a godoc

https://github.com/celestiaorg/celestia-core/pull/970/commits/b2c40618a975e9015276d0a9c07546d10d494738

the duplication is not ideal, but I usually prefer readme's since they're easier to read than comments or open up a new godocs window. 

perhaps in the future we could use something like https://github.com/posener/goreadme

Reply:
> perhaps in the future we could use something like https://github.com/posener/goreadme

I thought a tool like that might exist. I'm down to use that. It would be nice to align our documentation system with the other engineering teams so doc readers have a uniform experience